pom.xml

```diff
@@ -36,14 +36,10 @@
         <artifactId>eclipse-core-runtime</artifactId>
         <version>20070801</version>
       </dependency>
-      <dependency>
-        <groupId>xerces</groupId>
-        <artifactId>xercesImpl</artifactId>
-        <version>2.12.0</version>
-      </dependency>
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
+        <scope>test</scope>
```

Comment:
Is the scope useful in the `dependencyManagement` section? The scope is now duplicated here and in the actual dependency declaration below.

BTW, the other entries in the `dependencyManagement` could be updated as well, since both `html` and `eclipse-core-runtime` are not used anymore but we depend on other artifacts now.

Reply:
@joachimLengacher Is that the want that I told you to update? If yes, that this was a mistake. Didn't see that this was part of the `dependencyManagement` section. In this section, the scope is not necessary IMO but also not harmful.

Reply:
@daniel-kr yes, that's the one we talked about. I also missed that it is in the dependencyManagement section. I fixed it in a new PR: https://github.com/collaborationFactory/cplace-daisydiff/pull/2 Interestingly the schema seems to allow it there. At least the editor in Intellij didn't show a warning...

@csemrau I also removed the unused entries from dependencyManagement.

I'm not going to create a new release now as this shouldn't hurt in the one that I created yesterday. It will just go into the next one.

Reply:
> I'm not going to create a new release now as this shouldn't hurt in the one that I created yesterday. It will just go into the next one.

That's perfectly fine. 👍